UnityProject/Assets/Scripts/Items/Tool/AirlockPainter.cs

```diff
@@ -109,14 +110,8 @@ public void ServerPaintTheAirlock(GameObject paintableAirlock, GameObject perfor
 				}
 			}
 
-			DoorAnimatorV2 airlockAnim = paintableAirlock.GetComponent<DoorAnimatorV2>();
-
-			ServerChangeDoorBase(airlockAnim, paintJob);
-			ServerChangeOverlaySparks(airlockAnim, paintJob);
-			ServerChangeOverlayLights(airlockAnim, paintJob);
-			ServerChangeOverlayFill(airlockAnim, paintJob);
-			ServerChangeOverlayWeld(airlockAnim, paintJob);
-			ServerChangeOverlayHacking(airlockAnim, paintJob);
+			AirlockCatalogueSync airlockAnim = paintableAirlock.GetComponent<AirlockCatalogueSync>();
+			airlockAnim.index = currentPaintJobIndex;
```

Comment:
I think this would be better served as a method, that way it is more obvious to other developers that there is a cost to using this - that it does more than just setting a value.